sys-apps/dstat/files/dstat-0.7.4-fix-shebang.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/dstat b/dstat
+index 9359965..093c574 100755
+--- a/dstat
++++ b/dstat
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env python
++#!/usr/bin/python
```

Comment:
By the way, you can call `python_fix_shebang` https://dev.gentoo.org/~mgorny/python-guide/helper.html and no patches needed.

Reply:
ping

Reply:
It is actually wrapped properly. So there is no need for either the path or python_fix_shebang.

Sorry for causing confusion here.